spec/features/refunds/refund_defaults_applicant_feature_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.feature 'Refund Defaults - Applicant page', js: true do
```

Comment:
The new tests in rspec.  The steps have been replaced with methods, causing very long method names - but it was the easiest way